eng/testing/linker/project.csproj.template

```diff
@@ -6,6 +6,7 @@
     <RuntimeIdentifier>{RuntimeIdentifier}</RuntimeIdentifier>
     <RuntimePackDir>{RuntimePackDir}</RuntimePackDir>
     <TargetingPackDir>{TargetingPackDir}</TargetingPackDir>
+    <NETCoreAppMaximumVersion>99.0</NETCoreAppMaximumVersion>
```

Comment:
```suggestion
    <NETCoreAppMaximumVersion>{NetCoreAppMaximumVersion}</NETCoreAppMaximumVersion>
```

Reply:
and here an entry for it: https://github.com/dotnet/runtime/blob/19f97bd4d4440d09a8ff6a3efa7b4c9ea1207190/eng/testing/linker/trimmingTests.targets#L76. We try to avoid the 99.0 setting and use the current version instead.